core/trino-main/pom.xml

```diff
@@ -239,6 +239,11 @@
             <artifactId>trino-array</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
i implemented the arrow writing functionality in a new library, trino-arrow. but to use that as a spooling format, i need to include that in trino-main. is that reasonable?